functions/Install-DbaFirstResponderKit.ps1

```diff
@@ -113,32 +117,47 @@ Logs into server1\instance1 with SQL authentication and then installs the FRK in
 			Stop-Function -Message "Couldn't download the First Responder Kit. Download and install manually from https://github.com/BrentOzarULTD/SQL-Server-First-Responder-Kit/archive/master.zip." -ErrorRecord $_
 			return
 		}
-		
 	}
 	
-	PROCESS {
-		Foreach ($instance in $SqlInstance) {
+	process {
+		if (Test-FunctionInterrupt) { return }
+		
+		foreach ($instance in $SqlInstance) {
 			try {
-				Write-Host "Connecting to $instance"
-				$Connection = Connect-DbaSqlServer -SqlInstance $instance -Credential $sqlcredential
+				Write-Message -Level Verbose -Message "Connecting to $instance"
+				$server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
 			}
 			catch {
-				Write-Host "Failed to connect to $instance : $($_.Exception.Message)"
```

Comment:
We use a Stop-Function, which is an internal dbatools function but in general, it's better to use Write-Output in lieu of Write-Host https://stackoverflow.com/questions/19754069/powershell-difference-between-write-host-and-write-output